### PR TITLE
feat: form relayout according to figma

### PR DESCRIPTION
**Pull request by michelalbers**
### Open issues:

- [x] Figma layout suggests labels outside the form inputs - this collides with material design patterns
- [x] There is no file dropzone component out of the box, will be built within this branch
- [ ] Fix radii for boxes
- [ ] Remove Account Address from cancel button
- [ ] @mikeplanzer pls add feedback here

### CODE REVIEW

**Comment by michelalbers** [comment]
Ready for merge in my opinion - any objections?

**Comment by 2075** [comment]
please assign reviewers

**Comment by 2075** [comment]
also, there are conflicting files 👯 @michelalbers

**Comment by 2075** [comment]
image

**Comment by 2075** [comment]
image
maybe right aligned @mikeplanzer ?

**Comment by 2075** [comment]
looks good in general!

**Comment by 2075** [comment]
should these have a radius like the others as well?
image

**Comment by 2075** [comment]
if this is a dropzone, shall we let people know? @mikeplanzer 

image

**Comment by 2075** [comment]
can the imageboxes here get a small radius , too?
image

**Comment by 2075** [comment]
image
maybe check wording, not only logos here

File: src/themes/minimal/shape.ts [hunk 1]
```diff
@@ -8,9 +8,9 @@ declare module '@mui/system' {
 }
 
 const shape = {
-	borderRadius: 1,
-	borderRadiusSm: 1,
-	borderRadiusMd: 2,
+	borderRadius: 8,
```
Comment (2075) [inline src/themes/minimal/shape.ts:11]: suggesting a smaller radius for sm and md, how about 3 and 5?

File: src/themes/gamedao/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { createTheme } from '@mui/material/styles'
 
 const globalTheme = {
 	shape: {
-		borderRadius: 1,
+		borderRadius: '8px',
```
Comment (2075) [inline src/themes/gamedao/index.js:5]: shouldnt this be derived from your shapes props below?

**Review by 2075 — CHANGES_REQUESTED** [review]

**Comment by 2075** [comment]
sorry, more conflicts due to idle branches :)

File: src/themes/minimal/shape.ts (hunk 1 above)
Comment (michelalbers) [reply src/themes/minimal/shape.ts:11]: Just went with the one from figma. I'm very unopinionated about this in general - just tell me what you want to have :)

File: src/themes/gamedao/index.js (hunk 2 above)
Comment (michelalbers) [reply src/themes/gamedao/index.js:5]: I think this is not used anywhere anymore

**Comment by mikeplanzer** [comment]
Hey @michelalbers , good job so far! Please see my feedback:

1. Is it possible to not show the placeholder from the beginning? I would propose to only show a preview after an asset is uploaded.
2. We should add a placeholder text "Drop an image here, or select a file." so that users know they can do either. Also on an extra line an additional info text with images specs we need to define so that it works for the campaign page. Like "_It must be a JPG or PNG, 1920 x 800px and no larger than 200 MB._"
3. In light mode there is no border around the drop zone - same border color as for input fields can be used.

**Additional change request**
1. Can you change border to dashed instead of solid (Only for file upload section)
2. After the images is uploaded, the preview should be show within the dropzone
3. The uploaded image can be removed by clicking the X in the right corner
See Figma: https://www.figma.com/file/LmFOAGwKieWxmLtFVulChX/GameDAO-Frontend-Design?node-id=354%3A4522

Do we need an extra ticket for the additional requests?

**Comment by 2075** [comment]
@mike let's do new tickets for the updates, so we can merge here

**Comment by 2075** [comment]
@michelalbers kan i haz?

**Pull request merged**